README.md

```diff
@@ -7,15 +7,11 @@ If you installed the app through the package manager you only need to do step 5
 
 1. clone the github repository: https://github.com/ljo/exist-sparql.git
 
-2. You may then either build with Apache Maven (preferred) or Apache Ant.
-
-    1. To build with Apache Maven: `mvn clean compile package`
-
-    2. or, to build with Apache Ant: `ant`
+2. To build with Apache Maven: `mvn clean compile package`
 
 3. upload the xar file found in `target/` into eXist-db using the dashboard
 
-4. enable the Index module in conf.xml, by adding one module declaration under "indexer/modules":
+4. enable the Index module in `$EXIST_HOME/conf.xml`, by adding one module declaration under "indexer/modules":
 ```xml
 <module id="rdf-index" class="org.exist.indexing.rdf.TDBRDFIndex"/>
```

Comment:
After installing the xar, making this change to conf.xml, and restarting eXist 5.3.0-SNAPSHOT, I get this error in exist.log:

```
2020-06-25 12:09:04,712 [global.launcher.startJetty] ERROR (JettyStart.java [run]:216) - configuration error: element 'module' requires an attribute 'uri' 
org.exist.util.DatabaseConfigurationException: element 'module' requires an attribute 'uri'
	at org.exist.util.Configuration.loadModuleClasses(Configuration.java:433) ~[exist-core-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at org.exist.util.Configuration.configureXQuery(Configuration.java:390) ~[exist-core-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at org.exist.util.Configuration.<init>(Configuration.java:275) ~[exist-core-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at org.exist.util.SingleInstanceConfiguration.<init>(SingleInstanceConfiguration.java:56) ~[exist-core-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at org.exist.util.SingleInstanceConfiguration.<init>(SingleInstanceConfiguration.java:52) ~[exist-core-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at org.exist.jetty.JettyStart.run(JettyStart.java:199) ~[exist-core-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at org.exist.launcher.Launcher.lambda$4(Launcher.java:188) ~[exist-core-5.3.0-SNAPSHOT.jar:5.3.0-SNAPSHOT]
	at java.lang.Thread.run(Thread.java:834) [?:?]
```

Changing the `<module>` element to:

`<module uri="http://exist-db.org/xquery/sparql" class="org.exist.indexing.rdf.TDBRDFIndex"/>`

... allows eXist to start up without error.

Reply:
@joewiz there are two module lines - the first is an index module, and the second is a xquery module. It sounds like you may have placed the index module in the xquery module part of conf.xml.